slash-commands/slash-commands.js

```diff
@@ -67,15 +77,19 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 						var command = match[1];
 						var data = match[2];
 
+						// pass though some commands, they only merit formatting elsewhere
+						if ($.inArray(command, self.passthrough) != -1) { } 
 						// Match only whitelisted commands
-						if ($.inArray(command, self.commands) != -1) {
+						else if ($.inArray(command, self.commands) != -1) {
 							self[command](data);
-						} else {
-							// TODO: Better way to notify the user of the invalid command
-							alert("Invalid command: " + command);
+							args.message = '';
+						}
+						else {
```

Comment:
I'm seeing noting about /say in xep-0045 - and section 17.2 is what this plugin tries to fulfil. Preceding space works for me, and stripping is beyond the scope of this plugin.